moveit_ros/planning/planning_scene_monitor/include/moveit/planning_scene_monitor/planning_scene_monitor.h

```diff
@@ -113,7 +113,6 @@ class MOVEIT_PLANNING_SCENE_MONITOR_EXPORT PlanningSceneMonitor : private boost:
    *  @param name A name identifying this planning scene monitor
    */
   PlanningSceneMonitor(const rclcpp::Node::SharedPtr& node, const std::string& robot_description,
-                       const std::shared_ptr<tf2_ros::Buffer>& tf_buffer = std::shared_ptr<tf2_ros::Buffer>(),
                        const std::string& name = "");
```

Comment:
Would you mind creating deprecated constructors that just call the new ones that we can delete after some time?

Reply:
This (https://github.com/ros-planning/moveit2/pull/654#discussion_r719291506)